gratipay/models/participant.py

```diff
@@ -416,11 +417,11 @@ def add_email(self, email, resend_threshold='3 minutes'):
             nonce = self.db.one("""
                 UPDATE emails
                    SET verification_start=%s
-                 WHERE participant=%s
+                 WHERE participant_id=%s
                    AND address=%s
                    AND verified IS NULL
              RETURNING nonce
-            """, (verification_start, self.username, email))
+            """, (verification_start, self.id, email))
             if not nonce:
                 return self.add_email(email)
 
```

Comment:
Aha! I see. I applied your diff before running test. `participant` was still existing in `gratipay-test`. I had no idea `branch.sql` is run by the test suite. Figured it'd be like rails where the test database is created from the current schema.